modules/assim.sequential/R/sda.enkf.R

```diff
@@ -80,25 +82,42 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     sampleIDs <- c(1:n.inputs,sample.int(n.inputs, (nens - n.inputs), replace = TRUE))
   }
   
- 
+  # currently not focusing on met ensembles for ED-SDA coupling, and recycling same met if not localhost
+  # sample_met is a temporary function anyway 
   if(is.null(restart) & is.null(restart$ens.inputs)){
-    ens.inputs <- sample_met(settings,nens)
-  } else {
+    if(settings$host$name != "localhost"){ # temporary HACK..temporary HACK..temporary HACK
```

Comment:
can you make an issue for this, so we don't loose it. Maybe in that issue also specify what a solution is for this hack.

Reply:
I wish I could loose it 😛 

Let me see, I think this issue is relevant #1167 and also this #1455